app/models/message.rb

```diff
@@ -39,7 +39,6 @@ class Message < ApplicationRecord
   scope :outbox, -> { where(outbox: true) }
   scope :inbox, -> { where.not(outbox: true).where(type: nil).or(self.where.not(type: "MessageDraft")) }
 
-  after_update_commit ->(message) { EventBus.publish(:message_changed, message) }
```

Comment:
Hmm, ale ked tu toto bolo predtym, tak neviem uplne ci to chceme vyhodit, ci?

Reply:
Hej, pravda, radsej nechajme